src/value.c

```diff
@@ -181,6 +181,14 @@ inline bool SIValue_IsNull(SIValue v) {
 	return v.type == T_NULL;
 }
 
+inline bool SIValue_IsFalse(SIValue v) {
+	return !v.longval;
+}
+
+inline bool SIValue_IsTrue(SIValue v) {
+	return v.longval;
+}
```

Comment:
If think we need to assert on `v's` type. (`T_BOOL`).

Reply:
done